about.html

```diff
@@ -0,0 +1,138 @@
+            <li class="web-pages">
+              <a href="#">Join</a>
+            </li>
+            <li class="web-pages">
+              <a href="#">Sponsor</a>
+            </li>
+            <li class="web-pages">
+              <a href="#">News</a>
+            </li>
+          </ul>
+          <div class="company-name">
+            <a href="index.html">The NBA</a>
+          </div>
+        </nav>
+    </div>
+  </header>
+  <section class="first-section">
+    <div class="first-section-container">
+      <h1 class="open-heading">The NBA 2021 Work Shop</h1>
+      <div class="first-section-para">
+        <p>
+          The Global Summit brings together the community of experts, academics, and activists who comprise the Creative Commons affiliate network in a different country every two years. This year, we hope to expand our invitation list including organizations and invidivuals who want to work with us on shared projects that advance the cause of the Commons, free culture and open knowledge. So if you're active and engaged in the worlds of open content and knowledge - free software advocates, Wikipedians. Open Knowledge, galleries, libraries, museums, archives, governments and foundations, lawyers, and activists - we hope you'll consider joining us this year to build a stronger, more vibrant commons together.
+        </p>
+      </div>
+      <div class="event-details">
+        <p>Please contact us per email for any further questions
+          about the NBA Summit 2021!
+        </p>
+        <p class="email-address">
+          info@NBA.com
+        </p>
+      </div>
+    </div>
+  </section>
+
+  <section>
+    <div class="flex-content">
+      <h2> The NBA Summit 2021 MVP trophy</h2>
+      <hr>
+      <p class="mobile-para">The MVP of the NBA 2015 was decided through the NBA all star games from 8 June to 7 July.</p>
+      <p class='desktop-para'> The MVP of the NBA 2015 was decided through the NBA all star games from 8 June to 7 July.
+        After the online public voting and Selection Committee's judging process, the MVP from Milwakee bucks won the prize.
+        After consultation with Mr. Giannis we optimized the prize-winning-MVP for this year's summit as following:</p>
+    </div>
+    <div class="logo-container">
+      <img src="/assets/nbamvp.png" alt="">
+    </div>
+    <hr class="horizontal-line">
+      
+  </section>
+
+  <section class="past-section">
+    <div class="past-section-container flex-past">
+      <h2>See the past NBA Summit Games</h2>
+      <hr>
+      <p>Take a look at the last two NBA Global Summits which took place in New York and in Los Angelos.</p>
+      <img src="assets/m2.jpg" alt="">
+      <img src="assets/m11.jpg" alt="">
+    </div>
+  </section>
+
+  <footer>
+    <div class="footer-flex">
+    <div class="footer-logo">
+      <a href="#"> My Logo</a>
+    </div>
+    <p class="copy-right">
+      2021 NBA Summit Miami <br> SomeRights Reserved
+    </p>
+  </div>
+  </footer>
+
+
+
```

Comment:
- [ ] For best practices, please remove any extra blank lines or indentations for cleaner and more optimized code. 🙏 😄 
_You can find more about this tip and other useful ones at this [link](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)._